core-iconset.html

```diff
@@ -187,13 +202,13 @@
   },
 
   updateThemes: function () {
-    this.lightDom.querySelectorAll('property[theme]')
+    Polymer.dom.querySelectorAll('property[theme]', this)
```

Comment:
I'm gonna punt so hard on this given the opportunity ;) I have no desire to create a junk element, and I also have no context for why this feature even exists.